lib/manager/bazel/update.spec.ts

```diff
@@ -5,7 +5,7 @@ import { UpdateType } from '../../config';
 import { updateDependency } from './update';
 
 jest.mock('hasha');
-jest.mock('../../util/got');
+jest.mock('got');
```

Comment:
Why this is using got mock?

Reply:
For mocking `got.stream`, I'm figuring it out right now